src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -852,7 +852,7 @@ namespace NServiceBus
     {
         public Schedule() { }
     }
-    public class static ScheduleBusExtensions
+    public class static ScheduleExtensions
     {
         [System.ObsoleteAttribute("Please use `ScheduleEvery(this IBusContext context, TimeSpan timeSpan, Func<IBusC" +
```

Comment:
@mat-mcloughlin will you address these obsoletes in your PR? See for example mentioning of `IBusContext`